sos/policies/package_managers/__init__.py

```diff
@@ -158,32 +154,43 @@ def pkg_by_name(self, name):
         except Exception:
             return None
 
+    def _parse_pkg_list(self, pkg_list):
+        """
+        Using the output of `query_command`, build the _packages dict.
+
+        This should be overridden by distinct package managers and be a
+        generator for _generate_pkg_list which will insert the packages into
+        the _packages dict.
+
+        This method should yield a tuple of name, version, release for each
+        package parsed. If the package manager or distribution does not use a
+        release field, set it to None.
+
+        :param pkg_list: The output of the result of `query_command`
+        :type pkg_list:  ``str``
+        """
+        raise NotImplementedError
+
     def _generate_pkg_list(self):
         """Generates a dictionary of packages for internal use by the package
         manager in the format::
 
             {'package_name': {'name': 'package_name',
-                              'version': 'major.minor.version'}}
+                              'version': 'major.minor.version',
+                              'pkg_manager': 'package manager name'}}
```

Comment:
should we add the `release:`  so that we know that is part of the dict?

It though me off earlier today, as didn't realise that was part of the dict